apps/user-profile/src/components/modules/UserOnboardingModal/InterestBubble/InterestBubble.js

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from "react";
+
+import {
+  UserInterestCard,
+  UserInterestsCardContainer,
+  UserInterestsContainer,
+} from "./StyledInterestBubble";
+
+const UserInterests = () => {
+    const interests_list = [
+        {id:1, name:"Blockchain"}, {id:2, name:"Javascript"}, {id:3, name:"Web Design"}, 
+        {id:4, name:"UI/UX"}, {id:5, name:"React"}, {id:6, name:"QA"}, {id:7, name:"Web Development"}, 
+        {id:8, name:"C/C++/C#"}, {id:9, name:"Discord"}, {id:10, name:"Node.js"}, {id:11, name:"Strapi"}, 
+        {id:12, name:"Agile Development"}, {id:13, name:"HTML"}, {id:14, name:"Git"}, {id:15, name:"Frontend Development"}, 
+        {id:16, name:"Python"}, {id:17, name:"Data Structures"}, {id:18, name:"Prototyping"}, {id:19, name:"Project Management"}]
+    
+    const [selectedInterests, setSelectedInterests]=useState([])
+
+  return (
+    <UserInterestsContainer>
+          <div
+            style={{
```

Comment:
Move all styling to StyledInterestBubble, use classes instead. Keeps code cleans